src/supermin-init-prelude.sh

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env bash
+# This script is our half-baked ad-hoc reimplementation of a minimal
+# Linux boot environment that we generate via `supermin`.  At some
+# point we will likely switch to using systemd.
```

Comment:
(BTW, I think the best fix here is to generate a minimal initramfs that mounts the host (container) filesystem via virtiofs/9p.  Then we really don't need supermin at all, we can use systemd/ignition/etc. when we're building the ostree-based disk image, etc.)